tests/php/Dev/Tasks/SecureAssetsMigrationHelperTest.php

```diff
@@ -112,7 +112,7 @@ public function dataMigrate()
             ],
             'Unprotected' => [
                 'unprotected',
-                null,
+                '',
```

Comment:
Why has this changed?

Reply:
`TypeError: League\Flysystem\Filesystem::write(): Argument #2 ($contents) must be of type string, null given`

Method `write` in `LocalFilesystemAdapter class` expects to get string as 2nd param.